lib/widgets/app_theme.dart

```diff
@@ -1,13 +1,11 @@
 import 'package:flutter/material.dart';
-import 'package:cycle_lock/widgets/colours.dart';
+import 'package:cycle_lock/widgets/colors.dart';
 
 @immutable
 class AppTheme {
   static const colours = AppColours();
 
   const AppTheme._();
 
-  static ThemeData define() {
-    return ThemeData();
-  }
+  static ThemeData define() => ThemeData();
 }
```

Comment:
Can we can move this file to a separate theme folder as its not a widget

Reply:
Ok will do it and create themes folder inside views and migrate widgets folder inside views